docs/release-notes/8.16.asciidoc

```diff
@@ -56,6 +56,28 @@ On November 12, 2024, it was discovered that manually running a custom query rul
 ====
 // end::known-issue[]
 
+// tag::known-issue-53[]
+[discrete]
+.Alerts page crashes if you upgrade to 8.16 and access it in a non-default space  
+[%collapsible]
+====
+*Details* +
+On November 14, 2024, it was discovered that the Alerts page would crash if you upgraded to 8.16.0 and accessed the page in a non-default space.
+
+*Workaround* +
+Manually edit your browser's local storage and refresh the Alerts page:
+
+NOTE: These instructions only apply to the Google Chrome browser. Modify the steps based on of browser you're using.
+
+. Right-click anywhere on the Alerts page, then select *Inspect* to open Chrome's Developer Tools.
+. Go to *Application -> Storage*, then expand *Local Storage*. 
+. Click on the name of your Kibana instance, for example, http://localhost:1234. 
+. Search for the `siem.<space_name>.pageFilters` key and right-click on the value. 
```

Comment:
@michaelolo24 @logeekal, from reading the doc issue, it looks like multiple page filter keys can exist. Is this correct? If so, do you think it's worth mentioning in this step?

Reply:
Yes, at the end of everything here, maybe on line 77, I would state something to the effect:

`Be aware, that if you use multiple spaces in addition to default, this will need to be done for each non-default space`

Reply:
Thank you both. Agreed. Will add it to the knowledge base as well.

Reply:
Added in [5394c8d](https://github.com/elastic/security-docs/pull/6169/commits/5394c8d9acc1046ccb38113501760346bbef3ae5)